lte/protos/pipelined.proto

```diff
@@ -77,6 +77,7 @@ message RequestOriginType {
 message ActivateFlowsRequest {
   SubscriberID sid = 1;
   string ip_addr = 2; // Subscriber session ipv4 address
+  // Deprecated
   repeated string rule_ids = 3; // List of static rules obtained from PCRF
```

Comment:
should we rm `repeated string rule_ids = 3;` and add?
reserved 3;

Reply:
isn't it still used in pipelined though?

Reply:
yup good point will change there